test/System/MemoryInputStream.cpp

```diff
@@ -0,0 +1,31 @@
+#include <SFML/System/MemoryInputStream.hpp>
+#include "SystemUtil.hpp"
+
+#include <doctest.h>
+
+TEST_CASE("sf::MemoryInputStream class - [system]")
+{
+    SUBCASE("Empty stream")
+    {
+        sf::MemoryInputStream mis;
+
+        CHECK(mis.read(nullptr, 0) == -1);
+        CHECK(mis.seek(0) == -1);
+        CHECK(mis.tell() == -1);
+        CHECK(mis.getSize() == -1);
+    }
+
+    SUBCASE("Temporary memory stream")
+    {
+        const std::string memoryContents = "hello world";
+        sf::MemoryInputStream mis;
+        mis.open(memoryContents.data(), sizeof(char) * memoryContents.size());
+
+        char buffer[32];
+        CHECK(mis.read(buffer, 5));
```

Comment:
```suggestion
        CHECK(mis.read(buffer, 5) == 5);
```

Reply:
Should we not verify that the `mis.read()` actually returned the expected number of bytes read in this case?

Reply:
Yes, this is exactly what I suggested if I understand correctly.

Reply:
Oh wait yeah I misread, seems I'm reading my diffs wrong today. Yeah I'm perfectly happy with changing this.